src/SSRv1/static/editor.js

```diff
@@ -1207,9 +1207,11 @@
 
 		let delta = {};
 		let contents = [];
+		let importId = -1;
 
 		try {
 			counter = getNewFeaturesRecursively(root, delta, contents);
+			importId = root.children[1].children.namedItem("importBtn").value;
```

Comment:
Unless there's a specific reason for this to be in this try-catch, move it outside: it doesn't even handle its errors